app/models/event.rb

```diff
@@ -1,6 +1,7 @@
 class Event < ApplicationRecord
   has_many :volunteer_roles, dependent: :destroy
   validates :name, presence: true
+  validate :low_income_dates_valid?
```

Comment:
I think everything in this manual validator could be described with:

```
validates :low_income_requests_start, presence: true
validates :low_income_requests_end, presence: true, comparison: { greater_than: :low_income_requests_start }
```

Am I right?

Reply:
This doesn't seemingly work if they are both blank, which we entirely do allow to be the case. Whilst we always have a low income window, there should be the option to not

Reply:
```
validates :low_income_requests_start, presence: true, if: -> { low_income_requests_end.present? }
validates :low_income_requests_end, presence: true, comparison: { greater_than: :low_income_requests_start }, if: -> { low_income_requests_start.present? }
```

Reply:
@james updated now

Reply:
Grand, it works then?

Reply:
Yes it does